protobuf/closure_proto_js_library.bzl

```diff
@@ -0,0 +1,68 @@
+# -*- mode: python; -*-
+#
+# Copyright 2016 The Closure Rules Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Utilities for building JavaScript Protocol Buffers.
+"""
+
+load("//closure/compiler:closure_js_library.bzl", "closure_js_library")
+
+def closure_proto_js_library(
```

Comment:
[Optional] If you want to go for the gold, try defining this as a Skylark `rule`. If the rule returns `transitive_js_srcs`, `transitive_js_externs`, `js_language`, etc. in its `struct()` then it (sort of like duck typing) will be able to behave just like a `closure_js_library` rule from the perspective of parent rules.